packages/editor/src/lib/hooks/useCoarsePointer.ts

```diff
@@ -17,14 +17,25 @@ export function useCoarsePointer() {
 			return
 		}
 		if (window.matchMedia) {
-			const mql = window.matchMedia('(pointer: coarse)')
-			const handler = () => {
-				editor.updateInstanceState({ isCoarsePointer: !!mql.matches })
+			const mql = window.matchMedia('(any-pointer: coarse)')
+
+			const handler = (coarse?: boolean) => {
+				editor.updateInstanceState({ isCoarsePointer: coarse ?? !!mql.matches })
```

Comment:
We wouldn't want to update instance state on every pointer move / mouse move. I think we have logic now in our store that prevents us from updating state with its identical previous value, but I'd rather not even run that path.

Reply:
That makes sense, I'll work on that!